tests/base.py

```diff
@@ -29,6 +29,7 @@ class GoogleSheetsBaseTest(unittest.TestCase):
     START_DATE_FORMAT = "%Y-%m-%dT00:00:00Z"
     BOOKMARK_COMPARISON_FORMAT =  "%Y-%m-%dT%H:%M:%S.%fZ"
 
+    OBEYS_START_DATE = "obey-start-date"
```

Comment:
Please remove ref to start date feature. See test comment below.

Reply:
Removed.